src/struct_info.json

```diff
@@ -967,6 +967,13 @@
             "EM_FUNC_SIG_PARAM_F2I"
         ]
     },
+    {
+        "file": "emscripten/wasm_worker.h",
+        "defines": [
+            "ATOMICS_WAIT_NOT_EQUAL",
+            "ATOMICS_WAIT_TIMED_OUT"
```

Comment:
I am not a fan of this mechanism, since this represents a programming model that Emscripten users do not have. I.e. it shows practices that do not scale. That is why to me the bare literals with comments seems a preferable convention. But I guess since we don't have a great solution, works either way.

Reply:
I don't necessarily disagree, but I think we should be consistent.  If we thing this is wrong perhaps we should completely remove this mechanism.